cabal-install/bootstrap.sh

```diff
@@ -430,8 +432,9 @@ do_Cabal_pkg () {
     then
         if need_pkg "Cabal" ${CABAL_VER_REGEXP}
         then
+            ALEX_PATH="$(pwd)/$(find . -name alex -perm 755 | tail -n1)"
             echo "Cabal-${CABAL_VER} will be installed from the local Git clone."
-            (cd ../Cabal && install_pkg ${CABAL_VER} ${CABAL_VER_REGEXP})
+            (cd ../Cabal && ${ALEX_PATH} -g Distribution/C2Hs/Lexer.x && install_pkg ${CABAL_VER} ${CABAL_VER_REGEXP})
```

Comment:
I'm on the fence with these. The lexer for `.cabal` files is generated in manual preprocessing step (see `Makefile`). And rather keep it that way, i.e. let's not have compile-time dependency on alex.

Reply:
Will do.